webpack/InsightsCloudSync/InsightsCloudSync.js

```diff
@@ -2,25 +2,52 @@ import React from 'react';
 import { IntlProvider } from 'react-intl';
 import PropTypes from 'prop-types';
 import { translate as __ } from 'foremanReact/common/I18n';
-import { Button } from 'patternfly-react';
+import { Button, Icon } from 'patternfly-react';
 import { INSIGHTS_SYNC_PAGE_TITLE } from './InsightsCloudSyncConstants';
 
-const InsightsCloudSync = ({ syncInsights }) => {
+const InsightsCloudSync = ({ data: { settingsUrl }, syncInsights }) => {
   document.title = INSIGHTS_SYNC_PAGE_TITLE;
   return (
     <IntlProvider locale={navigator.language}>
       <div className="insights-cloud-sync">
         <h1>{__('Red Hat Insights Sync')}</h1>
-        <Button bsStyle="primary" onClick={syncInsights}>
-          {__('Sync now')}
-        </Button>
+        <p>
+          {__(`Insights synchronization process is used to provide Insights
+           recommendations output for hosts managed here`)}
+        </p>
+        <p>
+          {__(`1. Obtain an RHSM API token: `)}
+          <a
+            href="https://access.redhat.com/management/api"
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            access.redhat.com
+            <Icon name="external-link" />
+          </a>
+          <br />
+          {__("2. Copy the token to 'Red Hat Cloud token' setting: ")}
+          <a href={settingsUrl}>{__('Red Hat Cloud token')}</a>
```

Comment:
can we also open this link in another tab?
it can take time to find the right page again.